packages/odyssey-react/src/components/Select/Select/useChoices.tsx

```diff
@@ -35,9 +38,9 @@ const useChoices: UseChoices = ({
   const choices = useRef<undefined | Choices>();
```

Comment:
looks like this ref is no longer used and can be removed?

Reply:
It's used by the other effect that deals with change handling.

Reply:
oh - just realizing that if we never assign to the ref `current` property on line 40 below we've probably introduced a regression for the controlled component API when a developers need to set a specific value into the select (lines 104-115). Can you restructure your changes here to keep the ref assignment and then access the choices instance via `choices.current` ?

Reply:
As a FYI up we [merged the additional test coverage](https://github.com/okta/odyssey/pull/1182) for this interaction to `develop` here!